src/clr-base/utils/css-vars/css-vars.spec.ts

```diff
@@ -4,26 +4,25 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import * as ponyfill from './css-vars-ponyfill-fork.esm.js';
+import * as ponyfill from './css-vars-ponyfill-fork';
 import { runCssVarsPolyfill, cssVarsDefaultConfig } from './css-vars';
 
 describe('Utilities - CssVarsPolyfill ', () => {
   describe('runCssVarsPolyfill() ', () => {
     it('calls cssVars() with default config', () => {
-      const setPropertyTypeSpy = jasmine.createSpy('setPropertyTypeSpy');
-      spyOnProperty(ponyfill, 'cssVars').and.returnValue(setPropertyTypeSpy);
+      spyOn(ponyfill, 'cssVars');
       runCssVarsPolyfill();
-      expect(setPropertyTypeSpy).toHaveBeenCalledWith(cssVarsDefaultConfig);
+      expect(ponyfill.cssVars).toHaveBeenCalledWith(cssVarsDefaultConfig);
```

Comment:
Not sure why but after converting to TS the `spyOnProperty` would no longer work.

Reply:
Probably because it's being treated as a module. the SpyOnProperty was needed because of weirdness with jasmine when a file exported a function.